modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/facet/NestedFacetImpl.java

```diff
@@ -0,0 +1,107 @@
+/**
```

Comment:
When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation: